pwem/protocols/protocol_alignment_assign.py

```diff
@@ -47,6 +47,12 @@ def _defineParams(self, form):
         form.addParam('inputAlignment', params.PointerParam, pointerClass='SetOfParticles',
                       label="Input alignments",
                       help='Select the particles with alignment to be apply to the other particles.')
+        form.addParam('shiftsAppliedBefore', params.BooleanParam, default=False,
+                      label='Were particle shifts applied?',
+                      expertLevel=params.LEVEL_ADVANCED,
+                      help='This option should be set to Yes only if the particle shifts have been applied ' +
```

Comment:
I'd simplify a bit the help avoiding mentioning extraction: What about:
Activate this option only if the advanced option "Apply particle shifts" was set to Yes at the extract coordinates protocol.